test/e2e/policies.test-suite/balloons/n4c16/test06-update-configmap/code.var.sh

```diff
@@ -0,0 +1,77 @@
+# This test verifies that configuration updates via cri-resmgr-agent
+# are handled properly in the balloons policy.
+
+testns=e2e-balloons-test06
+
+cleanup() {
+    vm-command "kubectl delete pods --all --now; \
+        kubectl delete pods -n $testns --all --now; \
+        kubectl delete pods -n btype1ns0 --all --now; \
+        kubectl delete namespace $testns || :; \
+        kubectl delete namespace btype1ns0 || :"
+    terminate cri-resmgr
+    terminate cri-resmgr-agent
+    vm-command "cri-resmgr -reset-policy; cri-resmgr -reset-config"
+}
+
+apply-configmap() {
+    vm-put-file $(instantiate balloons-configmap.yaml) balloons-configmap.yaml
+    vm-command "cat balloons-configmap.yaml"
+    kubectl apply -f balloons-configmap.yaml
```

Comment:
Hmm... isn't this still wrong ? Shouldn't it be `vm-command "kubectl apply -f balloons-configmap.yaml"`

Reply:
Okay, I see. We have a `kubectl()` wrapper shell function defined in `run.sh`. I have to say, the way how we use both that and also `vm-command "kubectl..."`'s is totally super-confusing... We should at least try to be more consistent with these.